### PR TITLE
feat: Check Chat Tab regex for syntax errors

### PR DESCRIPTION
**Pull request by magicus**
If a previously stored regex is invalid, this is detected and the regex is ignored (but it is stored in the string, so if you edit the chat tab the old regex will be available for editing).

If an invalid regex is entered, the error message will be displayed, and the save button will be made unclickable.

### CODE REVIEW

**Comment by magicus** [comment]
This will solve #1318

File: common/src/main/java/com/wynntils/core/features/FeatureManager.java [hunk 1]
```diff
@@ -273,6 +273,7 @@ private void registerFeature(Feature feature) {
                 System.exit(1);
             }
         } catch (Throwable exception) {
+            // FIXME!!!
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/core/features/FeatureManager.java]: I assume a crashing init did fail to init all other features. I saw this behavior too.

**Review by kristofbolyai — APPROVED** [review]
LGTM (TextWidget now can be used in other screens)

File: common/src/main/java/com/wynntils/core/features/FeatureManager.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/core/features/FeatureManager.java]: Oops, I forgot this! That should not go in this PR; it was supposed to be a reminder to me to report this to the crash dump telemetry in a separate PR.

**Pull request merged**